### PR TITLE
Add two talks that are great for different reasons

### PR DESCRIPTION
**Pull request by dcmoore**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1262,6 +1263,7 @@ ever.
 * [It's not what you read, it's what you ignore](https://www.youtube.com/watch?v=IWPgUn8tL8s) [01:01:52] by **Scott Hanselman** (2012)
 * [WAT](https://www.destroyallsoftware.com/talks/wat) [00:04:17] by **Gary Bernhardt** (2012)
 * [Linus Torvalds on git](https://www.youtube.com/watch?v=4XpnKHJAok8) [01:10:15] by **Linus Torvalds** (2007) [A]
+* [Mob Programming A Whole Team Approach](https://vimeo.com/241193517) [00:37:25] by **Woody Zuill** (2017)
```
Comment (hellerve) [inline README.md]: This needs to go up a few lines (the list is ordered in reverse chronological order).

**Review by hellerve — COMMENTED** [review]
Hey! Thanks for contributing, I enjoyed both of these very much. <3

I have one minor comment, then this is ready to go!

File: README.md (hunk 1 above)
Comment (dcmoore) [reply README.md]: updated

**Review by hellerve — APPROVED** [review]

**Comment by hellerve** [comment]
Thank you!

**Pull request merged**